README.md

```diff
@@ -1,36 +1,100 @@
+   ```
+
+> Running `npm run dev` also executes `stepzen start`, which is the command to deploy your GraphQL API to the StepZen cloud.
+> You can as well run the `stepzen start` command at any point, in a serapate terminal window.
+>
+> Read more about what happens when you run `npm run dev`, in the ["Getting started" explained](#getting-started-explained) section below.
 
-```bash
-npm run dev
-# or
-yarn dev
-```
+---
 
-Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
+<img width="1306" alt="demo" src="https://user-images.githubusercontent.com/22416150/203358874-7ccbe260-0f70-4433-9340-974dd6e17616.png">
 
-You can start editing the page by modifying `pages/index.js`. The page auto-updates as you edit the file.
+Open [`http://localhost:3000`](http://localhost:3000) with your browser to see the app, and [`dashboard.stepzen.com`](https://dashboard.stepzen.com/) to see your GraphQL API endpoint.
 
-[API routes](https://nextjs.org/docs/api-routes/introduction) can be accessed on [http://localhost:3000/api/hello](http://localhost:3000/api/hello). This endpoint can be edited in `pages/api/hello.js`.
+You can edit the GraphQL API my modifying `stepzen/index.graphql` by hand, or use the [`stepzen import`](https://stepzen.com/docs/cli/cli-commands#stepzen-import) CLI command to import additional data sources to your API.
 
-The `pages/api` directory is mapped to `/api/*`. Files in this directory are treated as [API routes](https://nextjs.org/docs/api-routes/introduction) instead of React pages.
+The page auto-updates as you edit JS or CSS files, and the GraphQL API on `stepzen.net` auto-updates as you edit GraphQL schema files in `stepzen/`.
+
+The GraphiQL component on the home page is only an example to show that your app has a GraphQL API. Feel free to delete this component at any time. You can always use your [StepZen dashboard](https://dashboard.stepzen.com/) to explore your GraphQL endpoints.
 
 ## Learn More
 
-To learn more about Next.js, take a look at the following resources:
+To learn more about StepZen, take a look at the following resources:
 
-- [Next.js Documentation](https://nextjs.org/docs) - learn about Next.js features and API.
+- [StepZen Documentation](https://stepzen.com/docs) - learn about StepZen features and API.
 - [Learn Next.js](https://nextjs.org/learn) - an interactive Next.js tutorial.
 
-You can check out [the Next.js GitHub repository](https://github.com/vercel/next.js/) - your feedback and contributions are welcome!
+You can check out [the StepZen `examples` GitHub repository](https://github.com/stepzen-dev/examples) for more StepZen examples.
+
+## Deploy to Vercel
+
+If you created this app by clonning the starter repo with the "Deploy to Vercel" button, your Vercel project already has the `STEPZEN_ACCOUNT` and `STEPZEN_API_KEY` environment variables correctly defined. The StepZen Vercel integtaion does that for you, and you do not need to make any extra actions.
```

Comment:
"environment variables correctly defined" --> "environment variables correctly defined in your vercel project" with a link?

Reply:
https://vercel.com/docs/concepts/projects/environment-variables